modules/social_features/social_branding/modules/social_branding_required_test/social_branding_required_test.info.yml

```diff
@@ -0,0 +1,8 @@
+name: 'Social Branding required test'
```

Comment:
Added this sub-module just to test my custom hook. Used this as example: https://api.drupal.org/api/drupal/core%21modules%21system%21tests%21src%21Kernel%21System%21InfoAlterTest.php/class/InfoAlterTest/8.9.x

Reply:
Cool! Lets move it from `social_branding/modules/socail_branding_required_test` to `social_branding/tests/modules/<descriptive-name>`

You can see en example of this in the `locale` module in core. Moving this under the `tests` folder ensures it doesn't accidentally get enabled on production and hides it from the extensions page (it also prevents it from being discovered outside of tests).